lib/plugins/nextcloud-vue/rules/no-deprecated-exports.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+import type { Rule } from 'eslint'
+import { createLibVersionValidator } from '../utils/lib-version-parser.ts'
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+/*
+ Introduced in @nextcloud/vue v8.23.0
+ https://github.com/nextcloud-libraries/nextcloud-vue/pull/6385
+ */
+
+const rule: Rule.RuleModule = {
+	meta: {
+		docs: {
+			description: 'Deprecated @nextcloud/vue import syntax',
+			category: 'Nextcloud',
+			recommended: true,
+		},
+		fixable: 'code',
+		schema: [
+			{
+				// We accept one option which is an object
+				type: 'object',
+				properties: {
+					// if we should try to find an appinfo and only handle APIs removed before the max-version
+					parseAppInfo: {
+						type: 'boolean',
+					},
+					// Set a Nextcloud target version, only APIs removed before that versions are checked
+					targetVersion: {
+						type: 'string',
+					},
```

Comment:
But is it not related to the Nextcloud version or anything in the `appInfo`.
We only need to know the `@nextcloue/vue` version from the dependencies...